CMakeLists.txt

```diff
@@ -507,6 +507,11 @@ ELSE ()
 	ENDIF ()
 ENDIF()
 
+CHECK_FUNCTION_EXISTS(regcomp_l HAVE_REGCOMP_L)
```

Comment:
Should this use `CHECK_SYMBOL_EXISTS(regcomp_l "xlocale.h" HAVE_REGCOMP_L)` instead?